libavfilter/vf_libvmaf.c

```diff
@@ -1076,9 +1086,8 @@ const AVFilter ff_vf_libvmaf_cuda = {
 };
 #endif
 
-#if CONFIG_LIBVMAF_METADATA_FILTER
 const AVFilter ff_vf_libvmaf_metadata = {
-    .name           = "libvmaf_cuda",
+    .name           = "libvmaf_metadata",
```

Comment:
We don't need this extra filter at all. The only thing we need is to guard the option in the normal "libvmaf" filter.

Reply:
I couldn't understand, could you explain more? You want me do something like this; `libvmaf_filter_deps="libvmaf libvmaf-metadata"` 
instead of making 
`libvmaf_filter_deps="libvmaf"`
`libvmaf_metadata_filter_deps="libvmaf libvmaf_metadata"` in the `configure` ? If answer is yes, in this way `libvmaf` filter  doesn't work if not it's latest commit.

I couldn't find another way to do this in configuration stage without defining a new filter. If I not define a filter that macro (`CONFIG_LIBVMAF_METADATA_FILTER`) doesn't generate. I also tried with `VMAF_VERSION` but since it's string I can't do in macro stage.

Reply:
That was my mistake, somehow I missed `VMAF_VERSION_MAJOR/MINOR/PATCH`. Fixed in those commits.

https://github.com/yigithanyigit/FFmpeg/pull/2/commits/3ad9e15640089d3fef72089e986559faa9197382
https://github.com/yigithanyigit/FFmpeg/pull/2/commits/364b2194b511ad8a1f2a6597f04df5188fbf2d02